static/js/biomedical/disease/drug_table.tsx

```diff
@@ -1,17 +1,28 @@
 import React from "react";
 
 import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
-import { CompoundDiseaseTreatmentData } from "./chart";
+import {
+  CompoundDiseaseContraindicationData,
+  CompoundDiseaseTreatmentData,
+} from "./chart";
 
-export interface DrugTableProps {
+export interface CompoundDiseaseTreatmentProps {
   data: CompoundDiseaseTreatmentData[];
 }
 
-export function DrugTable(props: DrugTableProps): JSX.Element {
+export interface CompoundDiseaseContraindicationProps {
+  data: CompoundDiseaseContraindicationData[];
+}
+
+export function CompoundDiseaseTreatmentTable(
```

Comment:
Each file should only have one component. Please put the two different components into different files. Although, the two tables are pretty much exactly the same, so I think better, you should make one DrugTable component & use it for both the CompoundDiseaseTreatmentTable and the CompoundDiseaseContraindicationTable

Reply:
I think having one component makes more sense. But then, how would I change the props datatype and column names for different tables?

Reply:
we could have props take in a list of column headers and an array of objects with keys that match the headers eg.

```
interface DrugTableProps {
columns: string[];
data: { [columnKey: string]: string }[]
}
```
or can also make columns a map where key is a columnId and value is the display string.

Please also add comments for these props

Reply:
As per our discussion, I made column an array of objects and data, just a generic array of objects because it can belong to either the treatment type or contraindication type. Now that the indices are similar for the two objects, I was thinking we could just make column a simple array, instead of doing a name map. The current refactored code works for the two tables, but if there are any more suggestions, I'd be happy to implement them